src/common/src/hash/consistent_hash/vnode.rs

```diff
@@ -20,14 +20,15 @@ use crate::hash::HashCode;
 // TODO: make it a newtype
 pub type ParallelUnitId = u32;
 
-/// `VirtualNode` (a.k.a. VNode) is a minimal partition that a set of keys belong to. It is used for
+/// `VirtualNode` (a.k.a. Vnode) is a minimal partition that a set of keys belong to. It is used for
 /// consistent hashing.
 #[repr(transparent)]
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord, Display)]
 #[display("{0}")]
 pub struct VirtualNode(VirtualNodeInner);
 
-type VirtualNodeInner = u8;
+/// The internal representation of a virtual node.
+type VirtualNodeInner = u16;
```

Comment:
Would it be better to use newtype?

Reply:
We'll keep this type private, and the exported type `VirtualNode` is the newtype. 🥺